sklearn/preprocessing/data.py

```diff
@@ -1956,3 +1956,66 @@ def transform(self, X):
         """
         return _transform_selected(X, self._transform,
                                    self.categorical_features, copy=True)
+
+
+class CountFeaturizer(object):
```

Comment:
@nelson-liu Thank you for the feedback/guidance. I will add the things you mentioned in my next commit.